core/src/spotify_id.rs

```diff
@@ -24,10 +25,10 @@ impl SpotifyId {
             n = n + u128::from(d);
         }
```

Comment:
Instead of unwraping here it should return an error (ie the id has non-base16 characters)